perun-core/src/main/java/cz/metacentrum/perun/core/api/SecurityTeamsManager.java

```diff
@@ -125,12 +125,38 @@
 	 *
 	 * @param perunSession
 	 * @param securityTeam
+         * @param onlyDirectAdmins if true, get only direct user administrators (if false, get both direct and indirect)
 	 * @return list of users which are admis of given security team
 	 * @throws InternalErrorException
 	 * @throws PrivilegeException Can do only PerunAdmin or SecurityAdmin of the SecurityTeam
 	 * @throws SecurityTeamNotExistsException
 	 */
-	List<User> getAdmins(PerunSession perunSession, SecurityTeam securityTeam) throws InternalErrorException, PrivilegeException, SecurityTeamNotExistsException;
+	List<User> getAdmins(PerunSession perunSession, SecurityTeam securityTeam, boolean onlyDirectAdmins) throws InternalErrorException, PrivilegeException, SecurityTeamNotExistsException;
+
+        /**
+         * Gets list of direct user administrators of the SecurityTeam.
+         * 'Direct' means, there aren't included users, who are members of group administrators, in the returned list.
+         *
+         * @param perunSession
+         * @param securityTeam
+         * @return list of users which are admis of given security team
+         * @throws InternalErrorException
+         * @throws PrivilegeException
+         * @throws FacilityNotExistsException
+         */
+        @Deprecated
+        List<User> getDirectAdmins(PerunSession perunSession, SecurityTeam securityTeam) throws InternalErrorException, PrivilegeException, SecurityTeamNotExistsException;
```

Comment:
I don't think is good to create "deprecated" method from start. I think that getAdmins with "onlyDirectAdmins" parameter is enough.